packages/document/main-doc/docs/en/configure/app/plugins.mdx

```diff
@@ -9,39 +9,39 @@ sidebar_position: 9
 
 Used to configure custom Modern.js framework plugins.
 
-For the way to write custom plugins, please refer to [How to Write Plugins](/guides/topic-detail/framework-plugin/implement).
+Refer to [How to write plugins](/guides/topic-detail/framework-plugin/implement) for how to write custom plugins.
 
-## Precautions
+## Note
 
-This config **is used to configure the Modern.js framework plugin**. If you need to configure other types of plugins, please select the corresponding configs:
+This option is used to configure framework plugins. If you need to configure other types of plugins, please choose the corresponding configuration method:
 
-- Use [builderPlugins](/configure/app/builder-plugins) to configure Modern.js Builder plugins.
-- Use [tools.webpack](/configure/app/tools/webpack) or [tools.webpackChain](/configure/app/tools/webpack-chain) to configure webpack plugins.
-- Use [tools.babel](/configure/app/tools/babel) to configure Babel plugins.
+- To configure Modern.js Builder plugins, use the [builderPlugins](/configure/app/builder-plugins) option.
+- To configure webpack plugins, use the [tools.webpack](/configure/app/tools/webpack) or [tools.webpackChain](/configure/app/tools/webpack-chain) options.
+- To configure Babel plugins, use the [tools.babel](/configure/app/tools/babel) option.
 
-## Plugin type
+## Plugin types
 
-There are three different type of framework plugins built into the Modern.js:
+Modern.js has three types of built-in framework plugins:
```

Comment:
Modern.js has three types of built-in framework plugins 
->
Modern.js has three types of plugins